lib/utils/string_extension.dart

```diff
@@ -290,4 +290,12 @@ extension StringCasingExtension on String {
     );
     return result;
   }
+
+  String substringToHighlight(String highlightText, {int prefixLength = 0}) {
+    final index = toLowerCase().indexOf(highlightText.toLowerCase());
```

Comment:
should check the prefixLength < 0

Reply:
index also possible = -1, should check that case also

Reply:
i don't know what about in the case have icons ?

Reply:
done